src/app/core/signals/remote_signals/waku_backed_up_keypair.nim

```diff
@@ -0,0 +1,13 @@
+import json
+import base
+
+import ../../../../app_service/service/wallet_account/[keypair_dto]
+
+type WakuBackedUpKeypairSignal* = ref object of Signal
+  keypair*: KeypairDto
+
+proc fromEvent*(T: type WakuBackedUpKeypairSignal, event: JsonNode): WakuBackedUpKeypairSignal =
+  result = WakuBackedUpKeypairSignal()
+
+  if event["event"]{"backedUpKeypair"} != nil:
```

Comment:
Won't using `contains` show the intention better?

Reply:
@stefandunca thanks for the review. Updated.